packages/documentation-v7/src/stories/components/textarea/textarea.stories.ts

```diff
@@ -0,0 +1,262 @@
+      },
+    },
+    rows: {
+      name: 'Rows',
+      description:
+        'Attribute to set the initial height, in lines of text, of the `textarea` element.',
+      control: {
+        type: 'number',
+        min: 3,
+        max: 10,
+        step: 1,
+      },
+      table: {
+        category: 'General',
+      },
+    },
+    hint: {
+      name: 'Helper Text',
+      description: 'Text to place in the help text area of the component.',
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'General',
+      },
+    },
+    disabled: {
+      name: 'Disabled',
+      description:
+        'When set to `true`, disables the component\'s functionality and places it in a disabled state.<div className="mt-mini alert alert-info alert-sm">There are accessibility concerns with the disabled state.<br/>Please read our <a href="/?path=/docs/foundations-accessibility--page#disabled-state">disabled state accessibility guide</a>.</div>',
+      control: {
+        type: 'boolean',
+      },
+      table: {
+        category: 'States',
+      },
+    },
+    validation: {
+      name: 'Validation',
+      description: 'Controls the validation state appearance of the component.',
+      control: {
+        type: 'radio',
+        labels: {
+          'null': 'Default',
+          'is-valid': 'Valid',
+          'is-invalid': 'Invalid',
+        },
+      },
+      options: ['null', 'is-valid', 'is-invalid'],
+      table: {
+        category: 'States',
+      },
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+function renderTextarea(args: Args, context: StoryContext) {
+  const [_, updateArgs] = useArgs();
+  const id = `${context.viewMode}_${context.story.replace(/\s/g, '-')}_ExampleTextarea`;
+  const classes = mapClasses({
+    'form-control': true,
+    [args.size]: args.size !== 'null',
+    [args.validation]: args.valiation !== 'null',
+  });
+  const useAriaLabel = !args.floatingLabel && args.hiddenLabel;
+  const label = !useAriaLabel
+    ? html`
+        <label for=${id} class="form-label">${args.label}</label>
+      `
+    : null;
+  const contextuals = [
+    args.validation === 'is-valid'
+      ? html`
+          <div class="valid-feedback">Ggranda sukceso!</div>
+        `
+      : null,
+    args.validation === 'is-invalid'
+      ? html`
+          <div class="invalid-feedback">Eraro okazis!</div>
+        `
+      : null,
+    args.hint !== ''
+      ? html`
+          <div class="form-text">${args.hint}</div>
+        `
+      : null,
+  ];
+  const control = html`
+    <textarea
+      id=${id}
+      class=${classes}
+      defaultValue=${args.value ?? nothing}
+      rows=${args.rows}
+      ?disabled=${args.disabled}
+      aria-label=${useAriaLabel ? args.label : nothing}
+      aria-invalid=${VALIDATION_STATE_MAP[args.validation] ?? nothing}
+      @change=${(e: Event) => {
+        updateArgs({ value: (e.target as HTMLSelectElement).value });
+      }}
+    />
```

Comment:
Textarea is not a self-closing tag and if the closing tag is not on the same line as the opening tag, it contains a whitespace, which means, there is already a content. Like the user would have entered a space already.
This causes our floating-label to not work correctly, which is why we need to change the current line as follows:

```suggestion
    ></textarea>
```